babel.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = function(api) {
       ],
     ],
     plugins: [
+      'import-graphql',
```

Comment:
that methods we discussed aren't working, many parsing errors appearing. babel-plugin-import-graphql library makes it work correctly